frontend/src/features/admin-form/responses/FeedbackPage/FeedbackPage.tsx

```diff
@@ -211,40 +211,42 @@ export const FeedbackPage = (): JSX.Element => {
         }}
       >
         <Box gridArea="information" pl="0rem">
-          {currentFeedbackType === FeedbackType.Issues ? (
```

Comment:
issue: this change is not required 

explanation: since there are only 2 types and if the current feedback type is not issues, it will return the reviews. this pattern is repeated for the rest of the un-required changes.